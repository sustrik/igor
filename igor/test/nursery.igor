package foo

type X struct {
    n nursery
}

func foo(n nursery) nursery {
    var nn nursery
    nn.Close()
    return nil
}