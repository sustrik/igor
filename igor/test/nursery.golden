package foo

import __context "context"
import __igor "github.com/sustrik/igor/lib/igor"

type X struct {
	n __igor.Nursery
}

func foo(__ctx __context.Context, n __igor.Nursery) __igor.Nursery {
	var nn __igor.Nursery
	nn.Close(__ctx)
	return nil
}
